### PR TITLE
Introduce connection overview page

### LINKED ISSUE #2430: Connectivity overview page
**Describe the issue**
I got a question if there is an overview of which connections the ibis has (which queues, URLs etc.), but there is no such overview. To get such an overview, I would either need to check all adapters to check all listeners and senders. I think it would be useful to have a page where an overview of connections is listed.

**Reporter**
Ricardo

**To Reproduce**

**Screenshots**

**Configuration**
```
Please provide the configuration of the Pipe or Receiver with the problem.
```

**Input**
```
Please provide an example of the input message. 
Alternatively, provide a Ladybug report.
```

**Environment**
```
Please go to the console of your Frank, click on 'Information' and copy-paste the results here. 
```

**Additional Environment**
 - DBMS: [e.g. Oracle, MSSQL, MariaDB, MySql, PostgreSQL]
 - Browser: [e.g. Chrome, Safari, Edge, Firefox]

**Additional Context**
Add any other context about the problem here. (f.e. ladybug report / test adapter with larva test)

### PR DESCRIPTION
**Pull request by alisihab**
Closes #2430

### CODE REVIEW

**Comment by nielsm5** [comment]
Perhaps you can make the icon blue, nice arrow icon btw!
Overview looks good

**Review by nielsm5 — APPROVED** [review]

File: core/src/test/java/nl/nn/adapterframework/filesystem/MockFileSystemSenderTest.java [hunk 1]
```diff
@@ -10,10 +10,15 @@ protected IFileSystemTestHelper getFileSystemTestHelper() {
 
 	@Override
 	public FileSystemSender<MockFile, MockFileSystem<MockFile>> createFileSystemSender() {
-		FileSystemSender<MockFile,MockFileSystem<MockFile>> result=new FileSystemSender<MockFile,MockFileSystem<MockFile>>();
+		FileSystemSender<MockFile,MockFileSystem<MockFile>> result=new FileSystemSender<MockFile,MockFileSystem<MockFile>>(){
+
+			@Override
+			public String getDomain() {
+				return "";
+			}
```
Comment (gvanbrakel) [inline core/src/test/java/nl/nn/adapterframework/filesystem/MockFileSystemSenderTest.java:18]: This should not be necessary, the domain of the MockFileSystem should be used

File: core/src/test/java/nl/nn/adapterframework/filesystem/MockFileSystemPipeTest.java [hunk 2]
```diff
@@ -9,7 +9,12 @@ protected IFileSystemTestHelper getFileSystemTestHelper() {
 
 	@Override
 	public FileSystemPipe<MockFile, MockFileSystem<MockFile>> createFileSystemPipe() {
-		FileSystemPipe<MockFile,MockFileSystem<MockFile>> result = new FileSystemPipe<MockFile,MockFileSystem<MockFile>>() {};
+		FileSystemPipe<MockFile,MockFileSystem<MockFile>> result = new FileSystemPipe<MockFile,MockFileSystem<MockFile>>() {
+			@Override
+			public String getDomain() {
+				return "";
+			}
```
Comment (gvanbrakel) [inline core/src/test/java/nl/nn/adapterframework/filesystem/MockFileSystemPipeTest.java:16]: This should not be necessary, the domain of the MockFileSystem should be used

File: core/src/test/java/nl/nn/adapterframework/filesystem/MockFileSystemListenerTest.java [hunk 3]
```diff
@@ -16,7 +16,12 @@ protected IFileSystemTestHelperFullControl getFileSystemTestHelper() {
 			protected MockFileSystem<MockFile> createFileSystem() {
 				return ((MockFileSystemTestHelper<MockFile>)helper).getFileSystem();
 			}
-			
+
+			@Override
+			public String getDomain() {
+				return "";
+			}
```
Comment (gvanbrakel) [inline core/src/test/java/nl/nn/adapterframework/filesystem/MockFileSystemListenerTest.java:23]: This should not be necessary, the domain of the MockFileSystem should be used

**Review by gvanbrakel — CHANGES_REQUESTED** [review]

File: core/src/test/java/nl/nn/adapterframework/filesystem/MockFileSystem.java [hunk 4]
```diff
@@ -13,12 +13,13 @@
...
 import org.apache.logging.log4j.Logger;
 
+import lombok.Getter;
 import nl.nn.adapterframework.configuration.ConfigurationException;
 import nl.nn.adapterframework.stream.Message;
 import nl.nn.adapterframework.util.LogUtil;
 
 public class MockFileSystem<M extends MockFile> extends MockFolder implements IWritableFileSystem<M> {
-	
+	private final @Getter(onMethod = @__(@Override)) String domain = "Filesystem";
```
Comment (gvanbrakel) [inline core/src/test/java/nl/nn/adapterframework/filesystem/MockFileSystem.java:22]:
```suggestion
	private final @Getter(onMethod = @__(@Override)) String domain = "MockFilesystem";
```

File: core/src/test/java/nl/nn/adapterframework/filesystem/MockFileSystemSenderTest.java [hunk 5]
```diff
@@ -10,10 +10,8 @@ protected IFileSystemTestHelper getFileSystemTestHelper() {
 
 	@Override
 	public FileSystemSender<MockFile, MockFileSystem<MockFile>> createFileSystemSender() {
-		FileSystemSender<MockFile,MockFileSystem<MockFile>> result=new FileSystemSender<MockFile,MockFileSystem<MockFile>>();
+		FileSystemSender<MockFile,MockFileSystem<MockFile>> result=new FileSystemSender<MockFile,MockFileSystem<MockFile>>() {};
```
Comment (gvanbrakel) [inline core/src/test/java/nl/nn/adapterframework/filesystem/MockFileSystemSenderTest.java:13]:
```suggestion
		FileSystemSender<MockFile,MockFileSystem<MockFile>> result=new FileSystemSender<MockFile,MockFileSystem<MockFile>>();
```

**Review by gvanbrakel — CHANGES_REQUESTED** [review]

File: core/src/test/java/nl/nn/adapterframework/filesystem/MockFileSystemSenderTest.java (hunk 5 above)
Comment (alisihab) [reply core/src/test/java/nl/nn/adapterframework/filesystem/MockFileSystemSenderTest.java:13]: FileSystemSender is an abstract class

**Pull request merged**